proxy/nginx.conf.template

```diff
@@ -251,6 +253,23 @@ server {
     proxy_pass  $internalUrl/api/csp/csp-report;
   }
 
+  <% if ENV.key?("KEYCLOAK_URL") && ENV["KEYCLOAK_URL"] != "" %>
+
+  set $authUrl "<%= ENV["KEYCLOAK_URL"] %>";
+
+  location /auth {
+    <% if ENV["BASIC_AUTH_ENABLED"] == "true" %>
+    auth_basic "off";
+    <% end %>
```

Comment:
Why's this?

Reply:
Some functionality did not work with basic authentication enabled and seems to problem with Authorization header that both functionality use.

Reply:
Ah right, and everything in keycloak is always behind authentication? I.e. it's safe to leave "more open"?

Reply:
There is welcome page that is just links. But, we should not use dummy passwords in testing environments, but if/as we enforce password policy it should not be a problem. We could try to limit exceptions only to paths that are conflicting, but that would need to be tested with full flow.